### ISSUE TITLE
Arrange a temporary accessible space for when the lift is being replaced

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1833]
As we've had so many issues with the lift in the Tapestry the landlord is making a plan to replace it with a more suitable lift. This is great news as it should be a lot more reliable and may also mean that we are actually allowed to use it for transporting light goods (TBC). The bad part is that for a period of time while the new lift is being installed (potentially August-November), we will not have lift access to the first floor. We've been looking at options for taking some space somewhere else on a temporary basis to cover this period and have found one or two options. We need to have a better idea what should go in this space for that time period, partly as requested by the potential new landlords but also of course we need to have a plan for our own purposes.

We very much want to avoid "othering" the people who are likely to use this space. One way to do this might be to avoid just putting the bare minimum in, instead a selection of items could be moved into the space so that it is useful in its own right and becomes a place that anyone would want to use. Obviously there are some limits on this as the space will be smaller and we want to avoid a case whereby almost every job requires bouncing between the two spaces.

It would be best to assume that we won't have good ventilation options but we are intending to get a fume filtration system as mentioned in #1697 so we should be able to move a laser cutter. 3D printers and fabric arts related stuff should be safe, similarly electronics. I think we'd need to avoid the dustier stuff like woodworking though.

We would also hope to be able to host Maker Night partly in this space too which is why having relatively more equipment here would be good as we would want to make sure it was a place that everyone found useful, rather than just being a place for people who can't use the stairs.

For reference the best candidate location we have now is a room in C3 Liverpool on Kempston street, although the most accessible entrance will be from the back on Devon Street. It's not /so/ far if people do need to bounce between the spaces. I suspect we will struggle to get 24/7 access to that space although I'd hope that something similar to 9-5 + Maker Night/Day might be possible.

If anyone has other strong suggestions for locations we might use that would also be helpful, what we would be looking for would be:
 * Accessible from the street for people with mobility issues, e.g. wheelchair users
 * Accessible toilets for people with mobility issues
 * At least 25 square metres/250 square feet of space
 * Ideally 24/7 access that we can manage ourselves
 * Reasonably close to Kempston Street
 * Some ability to make refreshments, i.e. a small kitchen or at least some way we can fill a kettle

Please do make suggestions about equipment, alternative venues or anything else you think might be useful.

**Comment by defnetmedia** [comment]
You know this already but there is that new place opening downstairs that you could use. Not sure when their work will be finished but it is an option. You would probably still be able to use the same wifi too

**Comment by Robotorium** [comment]
That's a good point actually, would Sonny be willing to rent us some space
temporarily? It would cut down on the distance to move equipment/people,
we'd know the opening times, and we know it's accessible from Constance
Street.

Op do 15 jun. 2023 10:35 schreef defnetmedia ***@***.***>:

**Comment by johnmckerrell** [comment]
My understanding is that there wouldn't be anything spare on the ground floor but I've sent Sonny a message to confirm.

**Comment by amcewen** [comment]
(Not to distract from the focus on looking for a *temporary* space here, but just noting that if there's somewhere suitable that would only be available on longer-term basis, please share that anyway, as some longer-term extra space *could* be an option)

**Comment by johnmckerrell** [comment]
To update this issue, we've agreed to go ahead with taking a space in the C3 Liverpool building across the road. This has access via 2 steps from Kempston Street or wheelchair friendly access via the back of the building. There is also an accessible toilet on the same floor.

There is a kitchen that can be accessed by the named responsible people. I'm not sure if we raised the suggestion of bringing our own kettle and using it in the room but that might be an option.

There's likely to be some things we still need to work out before we can move in, not least of which they will need to move out the people currently using the room! We have pointed out that we need to move in ahead of the end of the month so we can be ready in advance of the lift being ready. I would say we can move in as soon as they're ready for us.

I'll attach a plan below but then add an additional comment as an undateable comment for planning what we need to move over.

![PNG image-66F9FEB0CE24-1](https://github.com/DoESLiverpool/somebody-should/assets/27243/f556e830-8d94-4b4b-b5e8-638d1762c0f0)

**Comment by johnmckerrell** [comment]
Actually rather than having an updateable comment here I've created a new issue for the actual move #1849 - I'll probably leave this one open until we've actually signed an agreement and have access (unless people find it too confusing having two places to look for information).

**Comment by zarino** [comment]
Quick update:

Sadly the landlord of the space in the C3 building is now no longer able to rent the space out.

@JackiePease and @Robotorium are investigating the possibility of renting the ground floor of the Thelma Madine Academy building, a few doors further up the street. We hope to have an update in the next few days.

**Comment by johnmckerrell** [comment]
We are now using the space in the yard which was previously the Melodic Distraction bar.
